services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerService.java

```diff
@@ -147,6 +148,11 @@ public boolean startInner() {
       admin.initStorageCluster(clusterName);
       consumerServicesByClusters.get(clusterName).start();
       LOGGER.info("started cluster: {}", clusterName);
+      if (!multiClusterConfigs.isParent()) {
+        for (HelixLiveInstanceMonitor liveInstanceMonitor: admin.getLiveInstanceMonitorList()) {
+          liveInstanceMonitor.refresh();
```

Comment:
I am wondering if this is the correct place to refresh.
I was not familiar with this part of code so I am asking here..it seems like the logic of re-enabling disabled partition was called in admin.initStroageCluster() which is before this part of logic. 
Could you explain a bit whether this can work?